.github/workflows/ci.yml

```diff
@@ -79,45 +108,66 @@ jobs:
-        echo "::set-output name=VERSION::$(<version)"
+    - name: Publish Release Assets
+      if: |
+        github.event.release
+      uses: actions/upload-release-asset@v1.0.2
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-    - name: Download all artifacts
-      if: github.event_name == 'push' && github.ref == 'refs/heads/master'
-      uses: actions/download-artifact@v2
-      with:
-        path: releases
-    - name: Verify Artifact Downloads
-      if: github.event_name == 'push' && github.ref == 'refs/heads/master'
-      run: ls -R
-    - name: Publish Release
-      if: github.event_name == 'push' && github.ref == 'refs/heads/master'
-      uses: softprops/action-gh-release@v1
       with:
-        tag_name: ${{ steps.release_ver.outputs.VERSION }}
-        files: releases/*/*.xz
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        upload_url: ${{ github.event.release.upload_url }}
+        asset_path: ./old/libraries-catalina-macos.tar.xz
+        asset_name: libraries-catalina-macos.tar.xz
+        asset_content_type: application/x-xz
+
+  # CreateRelease:
+  #   needs: [macOS, Docker_Linux]
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #   # - name: Increment Release Version
+  #   #   id: release_ver
+  #   #   if: github.event_name == 'push' && github.ref == 'refs/heads/master'
+  #   #   run: |
+  #   #     curl -H "Authorization: token ${GITHUB_TOKEN}" -s https://api.github.com/repos/${GITHUB_REPOSITORY}/releases/latest | jq -r '[.tag_name][0] | split(".") as $ver | $ver[-1]|tonumber as $last | $ver[:-1] as $first | $first + [$last+1] | map(tostring) |  join(".")' > version
+  #   #     echo "::set-output name=VERSION::$(<version)"
+  #   #   env:
+  #   #     GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  #   - name: Download all artifacts
+  #     # if: github.event_name == 'push' && github.ref == 'refs/heads/master'
+  #     if: github.event.release
+  #     uses: actions/download-artifact@v2
+  #     with:
+  #       path: releases
+  #   - name: Verify Artifact Downloads
+  #     # if: github.event_name == 'push' && github.ref == 'refs/heads/master'
+  #     if: github.event.release
+  #     run: ls -R
+  #   - name: Publish Release Assets
+  #     if: |
+  #       github.event.release
+  #     uses: actions/upload-release-asset@v1.0.2
+  #     env:
+  #       GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  #     with:
+  #       upload_url: ${{ github.event.release.upload_url }}
+  #       asset_path: ${{ env.ARTIFACT_NAME }}
+  #       asset_name: ${{ env.ARTIFACT_NAME }}
+  #       asset_content_type: application/x-xz
+  #   # - name: Publish Release
+  #   #   if: github.event_name == 'push' && github.ref == 'refs/heads/master'
+  #   #   uses: softprops/action-gh-release@v1
+  #   #   with:
+  #   #     tag_name: ${{ steps.release_ver.outputs.VERSION }}
+  #   #     files: releases/*/*.xz
+  #   #   env:
+  #   #     GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This is commented in favor of the `.github/workflows/vcpkg_release.yml` file, which relies on manual creation of tagged releases, and that file creates a `release` event that is now handled by all the different workflow files to publish the artifacts to the `github.event.release.upload_url`

Not sure if we _always_ want to tag a new release on every commit to `master`, since it seems a bit excessive.